### PR TITLE
Changelog v1.19.1

### PR DESCRIPTION
**Pull request by 6543**

### CODE REVIEW

File: CHANGELOG.md [hunk 1]
```diff
@@ -4,6 +4,93 @@ This changelog goes through all the changes that have been made in each release
...
+  * Remove conflicting CSS rules on notifications, improve notifications table (#23565) (#23621)
+  * Fix diff tree height and adjust target file style (#23616)
+* BUGFIXES
+  * Make label templates have consistent behavior and priority (#23749)
+  * Fix accidental overwriting of LDAP team memberships (#24050) (#24065)
+  * Fix branch protection priority (#24045) (#24061)
+  * Use actions job link as commit status URL instead of run link (#24023) (#24032)
+  * Add actions support to package auth verification (#23729) (#24028)
+  * Fix protected branch for API (#24013) (#24027)
+  * Fix: do not escape space between PyPI repository url and package name… (#23981) (#24008)
```
Comment (jolheiser) [inline CHANGELOG.md]:
```suggestion
  * Do not escape space between PyPI repository url and package name (#23981) (#24008)
```

**Comment by wxiaoguang** [comment]
Fix custom mailer template on Windows #24081

**Review by khmarbaise — APPROVED** [review]

**Review by silverwind — APPROVED** [review]

**Comment by silverwind** [comment]
Wait on https://github.com/go-gitea/gitea/pull/24083

**Comment by yardenshoham** [comment]
#24083 is merged

**Comment by 6543** [comment]
:robot:

**Comment by 6543** [comment]
-> #24092

**Pull request merged**